scripts/core/editor3/components/Editor3Component.tsx

```diff
@@ -278,13 +278,22 @@ export class Editor3Component extends React.Component<IProps, IState> {
         }
     }
 
+    isMacOS() {
+        if (navigator.userAgent.indexOf('Mac') != -1) {
+            return true;
+        }
+
+        return false;
+    }
+
     keyBindingFn(e) {
-        const {key, shiftKey, ctrlKey} = e;
+        const {key, shiftKey, ctrlKey, metaKey} = e;
         const selectionState = this.props.editorState.getSelection();
+        const osBasedKeyRule = this.isMacOS() ? metaKey : ctrlKey;
```

Comment:
keyboardModifier, mainModifier, modifierKey or modifier would be better choices for naming

Reply:
But this isn't some random naming, I'm taking `metaKey` from the event. If the platform is macOS and `metaKey` (cmd on Mac) is pressed then I have to use that key. I'm not sure whether we can use just any name here